api/config.yaml

```diff
@@ -227,3 +227,26 @@ yarnaudit:
   language: JavaScript
   default: true
   timeOutInSeconds: 360
+
+gitleaks:
+  name: gitleaks
+  image: huskyci/gitleaks
+  imageTag: "2.1.0"
+  cmd: |+
+    mkdir -p ~/.ssh &&
+    echo 'GIT_PRIVATE_SSH_KEY' > ~/.ssh/huskyci_id_rsa &&
+    chmod 600 ~/.ssh/huskyci_id_rsa &&
+    echo "IdentityFile ~/.ssh/huskyci_id_rsa" >> /etc/ssh/ssh_config &&
+    echo "StrictHostKeyChecking no" >> /etc/ssh/ssh_config &&
+    cd src
+    git clone -b %GIT_BRANCH% --single-branch %GIT_REPO% code --quiet 2> /tmp/errorGitCloneGitleaks
+    if [ $? -eq 0 ]; then
+      $(which gitleaks) --log=warn --report=results.json --repo-path=./code 2> /dev/null
```

Comment:
Maybe we should consider this error here and not send it to `2> /dev/null`. By doing so, we could debug any `gitleaks` errors smoother. What do you think of handling this error and it occurs? We could use `ERROR_RUNNING` as the output of the container. 🙃

Reply:
sound like a good idea, I'll try to update it